README.md

```diff
@@ -11,35 +11,70 @@ While this library is fully open source and HashiCorp will be maintaining it (si
 
 # Usage
 
-An Event is a collection of data, analogous to a log entry, that we want to process in a Graph.  The client provides an event type and payload, and any other fields are generated as part of processing. The library will not attempt to discover whether configured formatter/marshaller nodes can actually handle the arbitrary payloads; it is up to the encapsulating program to put any such constraints on the user via its API.
-
-The library's clients submit events to a Broker that routes them through a Graph based on their type.  A Graph is composed of Nodes.  A Node processes an Event in some way -- modifying it, filtering it, persisting it, etc.  A Sink is a Node that persists an Event.
+An Event is a collection of data, analogous to a log entry, that we want to
+process via pipelines. A pipeline is a graph composed of nodes.  The client
+provides an event type and payload, and any other fields are generated as part
+of processing. The library will not attempt to discover whether configured
+formatter/marshaller nodes can actually handle the arbitrary payloads; it is up
+to the encapsulating  program to put any such constraints on the user via its API.
+
+The library's clients submit events to a Broker that routes them through 
+pipelines (graphs) based on their type.  A pipeline is a graph composed of
+Nodes.  A Node processes an Event in some way -- modifying it, filtering it,
+persisting it, etc.  A Sink is a Node that persists an Event.
 
 ## Broker
 
-Clients interact with the library via the Broker.
-
-A Broker processes incoming Events, by sending them to the Graph associated with the Event's type.  A given Broker, along with its associated set of Graphs, will be configured programmatically. 
+Clients interact with the library via the Broker by sending events. A Broker
+processes incoming Events, by sending them to the pipelines (graphs) associated
+with the Event's type.  A given Broker, along with its associated set of
+pipelines (graphs), will be configured programmatically. 
 
 
 ## Nodes 
 
-A Node is a node in a Pipeline, that can perform any kind of operation that it wants to on an Event.  A node has a Type, one of: Processor, Formatter, Sink.
+A Node is a node in a Pipeline, that can perform operations on an Event.  A node
```

Comment:
Rereading this I realize it doesn't really help a lot.
But I don't know how to rephrase it.
Maybe we should copy that diagram from the RFC?

Reply:
Hum... the pic would make things a bit clearer.

Reply:
I'm adding a pic.